presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -2068,18 +2100,38 @@ public void run(ExtendedHiveMetastore metastore)
                     // Not deleting the table may leave garbage behind. The former is much more dangerous than the latter.
                     // Therefore, the table is not considered added.
                 }
+
                 if (!done) {
                     throw e;
                 }
             }
+            tableCreated = true;
+        }
+
+        private boolean hasTheSameSchema(Table newTable, Table existingTable)
+        {
+            List<Column> newTableColumns = newTable.getDataColumns();
+            List<Column> existingTableColumns = existingTable.getDataColumns();
+            // we only check for the name and type of the columns
```

Comment:
This comment is out of place for the Hive connector. Probably just delete it? Or mention why we only validate name and type (it's fine if column comments are different because queries won't fail)?